java-JDBC/jdbc-intro/JDBC-intro_checklist.md

```diff
@@ -98,3 +98,21 @@
        Good:
         SELECT * FROM manufacturers WHERE id = 1;
 ```           
+* When you convert `ResultSet` to `Manufacturer` create an object using setters or constructor but not both of them.
```

Comment:
```suggestion
* When you convert `ResultSet` to `Manufacturer` better create an object using setters or constructor but not both of them, because....
```